src/views/Publish.vue

```diff
@@ -1,9 +1,10 @@
 <template>
     <div class="view publish">
         <Navigator :onBack="() => this.$router.push( { name: 'Story' } )" />
-        <div class="header">
-            <h2 class="title" v-html="storyInfo.title"></h2>
+        <div :class="`header ${isFormError?'error':''}`">
```

Comment:
Did you know that `:class` also accept an object like `{ class: <condition> }`. For example: `:class="{ header: true, error: isFormError }"`